lib/generators/react_on_rails/templates/base/base/client/package.json.tt

```diff
@@ -71,11 +71,11 @@
     <%- if options.redux? -%>
     "mirror-creator": "1.1.0",
     <%- end -%>
-    "react": "^0.14.8",
+    "react": "^15.0.0",
```

Comment:
@brucek We need to change the peer dependency of the npm module to support either 14 or 15.